index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+<head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    <title>Tufoins portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+
+
+</head>
+
+
+<body>
+
+    <section class="toolbar">
+        <header>
+            <!--<nav class="navbar">-->
+                <ul>
+                    <li class="logo"> <a href="logo-link" ></a>Tufoin's logo</a></li>
```

Comment:
- [ ] Kindly remove one closing tag of `a` in this line. If you see `<a href="logo-link" ></a>Tufoin's logo</a>`, there you closed it and after text, you again used close tag to `a`. And also put your text inside the `a` tag then it will make sense when you navigate or remove the `a` close tag before text 👍.